apps/condo/domains/organization/schema/FindOrganizationsByTinService.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Generated by `createservice organization.FindOrganizationsByTinService --type queries`
+ */
+
+const get = require('lodash/get')
+
+const { featureToggleManager } = require('@open-condo/featureflags/featureToggleManager')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const { COMMON_ERRORS } = require('@condo/domains/common/constants/errors')
+const { USER_WHITE_LIST_FOR_FIND_ORGANIZATIONS_BY_TIN } = require('@condo/domains/common/constants/featureflags')
+const access = require('@condo/domains/organization/access/FindOrganizationsByTinService')
+const { FindOrganizationsByTinLog } = require('@condo/domains/organization/utils/serverSchema')
+const { STAFF } = require('@condo/domains/user/constants/common')
+const {
+    checkDailyRequestLimitCountersByUser,
+    checkTotalRequestLimitCountersByUser,
+} = require('@condo/domains/user/utils/serverSchema/requestLimitHelpers')
+
+
+const appLogger = getLogger('condo')
+const logger = appLogger.child({ module: 'organization/findOrganizationsByTin' })
+
+const MAX_TOTAL_REQUESTS = 50
+const UNAVAILABLE_TINS = ['0000000000', '000000000000']
```

Comment:
Other countries might have other TINs... Or these tins might fail validation rules in other countries.. 

Maybe move it to environment variables? Or better yet, create a list of reserved TINs by locale and use it in validation logic

Reply:
Search for organizations does not depend on the locale, you can search for any TIN

Reply:
Added envs to specify your own values